app/controllers/pages_controller.rb

```diff
@@ -1,11 +1,9 @@
 class PagesController < BaseController
 
+  before_filter :poll_authentication, only: [:login, :signup]
```

Comment:
Don't worry too much about this, but if you're curious search up `before_filter`s in rails controllers!